ci-operator/step-registry/ovn/sdn-migration/ovn-sdn-migration-commands.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET=${TARGET:-OVNKubernetes}
+
+oc get co
+
+oc patch Network.operator.openshift.io cluster --type='merge'   --patch '{"spec":{"migration":null}}'
+sleep 10
+
+# Change network to target network in Network.operator.openshift.io the CR to trigger machine config update by MCO.
+oc patch Network.operator.openshift.io cluster --type='merge' --patch "{\"spec\":{\"migration\":{\"networkType\":\"${TARGET}\"}}}"
+# Wait until MCO starts applying new machine config to nodes
+sleep 120
```

Comment:
don't think we need it?

Reply:
what kind of changes and propagated where? is there something we can check on in a loop until we know it's ready?